wagtail/wagtailadmin/templates/wagtailadmin/404.html

```diff
@@ -0,0 +1,34 @@
+{% extends "wagtailadmin/admin_base.html" %}
+{% load i18n staticfiles wagtailadmin_tags %}
+{% block titletag %}{% trans "Error 404: Page not found" %}{% endblock %}
+
+
+{% block extra_css %}
+    {{ block.super }}
+
+    <link rel="stylesheet" href="{% static 'wagtailadmin/css/layouts/404.css' %}" type="text/css" />
+{% endblock %}
+
+
+{% block furniture %}
+    <div class="page404__bg">
+        <div class="wagtail-logo-container__desktop u-hidden@small">
+          <img class="wagtail-logo wagtail-logo__body" src="{% static 'wagtailadmin/images/logo-body.svg' %}" alt=""/>
+          <img class="wagtail-logo wagtail-logo__tail" src="{% static 'wagtailadmin/images/logo-tail.svg' %}" alt="" />
+          <img class="wagtail-logo wagtail-logo__eye--open" src="{% static 'wagtailadmin/images/logo-eyeopen.svg' %}" alt="" />
+          <img class="wagtail-logo wagtail-logo__eye--closed" src="{% static 'wagtailadmin/images/logo-eyeclosed.svg' %}" alt="" />
+        </div>
```

Comment:
From what I can see this is the same code as the logo in the admin menu within the main admin screens. Could you extract this to a separate template file and reuse it in both places so we don't duplicate code?